src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,261 @@
+    tag: tag,
+    includeTriggered: true,
+  });
+
+  if (Array.isArray(notifications)) {
+    notifications.forEach((notification) => notification.close());
+  }
+};
+```
+
+### Debugging
+
+You can use Chrome DevTools' Notifications debug service on the Application tab. To start debugging,
+press the small "Start recording events" button in the top-left corner (the circle), or hit
+'Command/Control' + 'E'. This allows you to both examine scheduled notifications, as well as see
+displayed and closed notifications. The screenshots below show this in action.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="devtools-scheduled.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Chrome DevTools showing the "scheduled" state of a notification.
+  </figcaption>
+</figure>
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="devtools-displayed.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 2 — Chrome DevTools showing the "displayed" state of a previously scheduled notification.
+  </figcaption>
+</figure>
+
+### Demo
+
+You can see Notification Triggers in action in the [demo][demo], which allows you to schedule
+notifications, list scheduled notifications, and cancel them. The source code is available on
+[Glitch][demo-source].
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="demo.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 3 — Notification Triggers <a href="https://notification-triggers.glitch.me/">demo</a>.
+  </figcaption>
+</figure>
+
+## Security and permissions
+
+We've designed and implemented Notification Triggers using the core principles defined in
+[Controlling Access to Powerful Web Platform Features][powerful-apis], including user control,
+transparency, and ergonomics. As this API depends on service workers, it is only available in secure
+contexts. Using the API requires the same permission as regular push notifications.
+
+### User control
+
+This API is only available in the context of a `ServiceWorkerRegistration`. This implies that all
+required data is stored in the same context and is automatically deleted when the service worker is
+deleted or the user deletes all site data for the origin. Blocking cookies also prevents service
+workers from being installed, and therefore this API from being used. Notifications can always be
+disabled for the site in site settings.
+
+### Transparency
+
+Unlike the Push API, this API does not depend on the network, which implies scheduled notifications
+need all required data beforehand, including fetching image resources referenced by the `badge`,
+`icon` and `image` attributes. This means showing a scheduled notification is not observable by the
+developer and doesn't involve waking up the service worker until the user interacts with the
+notification. Consequently there is currently no known way the developer could obtain information
```

Comment:
There should be a comma after "Consequently".

Reply:
Thanks.